kronolith/lib/Event.php

```diff
@@ -775,7 +782,9 @@ public function toiCalendar($calendar)
         $vEvent->setAttribute('SUMMARY', $this->getTitle());
 
         // Organizer
-        if (count($this->attendees)) {
+        if ($this->organizer) {
+            $vEvent->setAttribute('ORGANIZER', 'mailto:' . $this->organizer, array());
+        } elseif (count($this->attendees)) {
```

Comment:
This conditional doesn't add attendees, it adds the current Horde user as an ORGANIZER if it is a "meeting", as determined by the fact that there are other attendees.